src/components/BodySchema/BodySchema.js

```diff
@@ -31,11 +31,13 @@ export default class BodySchema extends Component {
     }
 
     const { expandedProps } = this.state
+    const hasNoAdditionalProperties = this.props.additionalProperties === false
 
     return (
       <table className={classNames(classes.bodySchema, classes[styleVariation], {
         [classes.hasTabs]: hasTabs
       })}>
+        {hasNoAdditionalProperties && <caption className='isAdditional'>These are the only possible properties:</caption>}
```

Comment:
As you mentioned, need a little love with styling here (left align is a pretty good start, might not need much more!)